ts/components/core/selection/RadioButtonList.tsx

```diff
@@ -52,7 +52,13 @@ export const RadioButtonList = <T extends any>(props: Props<T>) => (
     )}
     <View style={styles.itemsContainer}>
       {props.items.map((item, key) => (
-        <View key={`radio_item_${key}`} style={styles.item}>
+        <View
+          key={`radio_item_${key}`}
+          style={styles.item}
+          accessible={true}
+          accessibilityLabel={item.label}
+          accessibilityRole="radio"
```

Comment:
As you showed in the screenshot the role is not interpreted correctly (I have not found anything about it but I believe it is a react native bug related to this specific role). 
In fact it is shown as `value` while it should be shown as `type`.
This changes the reading order of screen readers not following the standard.

Maybe we could compose the role by hanging it at the end of the label, what do you think?

Reply:
Got it, sorry I haven't noticed before. I'll remove this update and start an investigation in parallel to understand what's wrong with that.
Rolled back https://github.com/pagopa/io-app/pull/3288/commits/a4630ff89e8c44f21ebe2a43bd1c9b0f82bd6534